committeeoversight/urls.py

```diff
@@ -25,7 +26,11 @@
 
 from committeeoversightapp.views import pong
 
+def trigger_error(request):
+    division_by_zero = 1 / 0
+
 urlpatterns = [
+    path('sentry-debug/', trigger_error),
```

Comment:
do you want to keep this phony route in tact? if no, i'd support deleting it, plus the related function!

Reply:
sure!